src/tests/run.proj

```diff
@@ -337,6 +340,16 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
                         output.WriteLine(line)%3B
                     }
 
+                    // Add Android app running log to testOutput
+                    if ("$(TargetOS)" == "Android")
+                    {
+                        string androidLogFile = System.IO.Path.GetFullPath(_Global.coreRoot + "/AndroidApps" + "/" + _Global.nameSpace + "/adb-logcat-net.dot." + _Global.nameSpace + ".log")%3B
+                        if(File.Exists(androidLogFile))
+                        {
+                            testOutput.AddRange(System.IO.File.ReadAllLines(androidLogFile))%3B
```

Comment:
I think we should only add the android log if the test failes, it's not really useful for a successful run and just bloats the results xml.

You can extract the `ret == CoreclrTestWrapperLib.EXIT_SUCCESS_CODE` a few lines below into a `success` variable and only add the log file if it's false.

Reply:
Fixed it. However, ideally, I think the log should be handled by xharness.